7_react/react2/week1/github_user_searcher/src/App.jsx

```diff
@@ -0,0 +1,20 @@
+import SearchInput from "./components/SearchInput";
+import "./App.css";
+import { GithubSearchProvider } from "./context/GithubSearchContext";
+import SearchResult from "./components/SearchResult";
+
+function App() {
+  return (
+    <>
+      <div className="app-container">
+        <h1>Github user searcher</h1>
+        <GithubSearchProvider>
```

Comment:
alternatively (its more common) you can wrap the whole app by the provider, but this is also fine :)